tiktoken/load.py

```diff
@@ -44,7 +44,9 @@ def read_file_cached(blobpath: str) -> bytes:
     if os.path.exists(cache_path):
         with open(cache_path, "rb") as f:
             return f.read()
-
+    elif os.environ.get("TIKTOKEN_FORCE_CACHE") == "1":
```

Comment:
or `TIKTOKEN_NO_DOWNLOAD`?

Reply:
Yes, why not. I will let the maintainers decide.

Reply:
From my pov, this should be passed via the library API not an os.environ.

Reply:
It was to keep the spirit of the function that uses `os.environ` to retrieve the cache_dir